app/src/main/java/com/example/poste/models/Post.java

```diff
@@ -95,11 +95,7 @@ public String getUrl() {
     public String getId() { return id; }
 
     public ArrayList<String> getTags() {
-        ArrayList<String> ret = new ArrayList<>();
-        for (String s : tags) {
-            ret.add(s);
-        }
-        return ret;
+        return new ArrayList<>(tags);
```

Comment:
Really, we could probably just return tags, but in sticking with the original intent of returning a copy, I just cleaned it up a bit. No need to parse a list into strings to create a new list